doc/periodictask/eventcounter.rst

```diff
@@ -3,4 +3,39 @@
 EventCounter
 ------------
 
-TODO
\ No newline at end of file
+The Event Counter task module can be used with the :ref:`periodic_tasks` to create time series of certain events.
+An event could be a failed authentication request. Using the Event Counter privacyIDEA can create graphs that display
+the development of failed authentication requests over time.
+
+To do this, the Event Counter task module reads a counter value from the database table ``EventCounter`` and writes this
+current value in the time series in the database table ``MonitoringStats``.
```

Comment:
...current value **as a** time series...

Reply:
The current value is not written as a time series.
I think the current value is added to a time series? (as the time series consists of a lot of current values)?